tests/cpputest/wrapTestModels.m

```diff
@@ -4,20 +4,38 @@
     
     amiciPath = fileparts(mfilename('fullpath'));
     amiciPath = [amiciPath '/../..'];
-
+    
     %% EXAMPLE STEADYSTATE
-
+    
     cd([amiciPath '/examples/example_steadystate/']);
-
-    [exdir,~,~]=fileparts(which('example_steadystate.m'));
-    amiwrap('model_steadystate','model_steadystate_syms',exdir)
-          
+    
+    try
+        [exdir,~,~]=fileparts(which('example_steadystate.m'));
+        amiwrap('model_steadystate','model_steadystate_syms',exdir);
+    catch
+        cd(fileparts(mfilename('fullpath')));
```

Comment:
The 'catch' was only meant to change back to the original working directory, no real error handling is taking place. The problem throwing the exception in the first place will probably persist.